src/libstd/thread.rs

```diff
@@ -283,7 +283,7 @@ impl Builder {
         // address at which our stack started).
         let main = move || {
             let something_around_the_top_of_the_stack = 1;
-            let addr = &something_around_the_top_of_the_stack as *const isize;
+            let addr = &something_around_the_top_of_the_stack as *const i32;
```

Comment:
It's irrelevant what `something_around_the_top_of_the_stack` is, so `i32` is just okay.